trikot-foundation/trikotFoundation/src/nativeMain/kotlin/com/mirego/trikot/foundation/concurrent/AtomicReference.kt

```diff
@@ -1,45 +1,13 @@
 package com.mirego.trikot.foundation.concurrent
 
-import kotlin.native.concurrent.freeze
-import kotlin.native.concurrent.isFrozen
-
 actual class AtomicReference<T> actual constructor(value: T) {
```

Comment:
This should be the baseline implementation for all targets. And eventually we can deprecate it in favor of using atomic-fo directly.